README.md

```diff
@@ -1,29 +1,38 @@
- 
 Obviously, you can only reconstitute complete original FASTQs (except for order), if the BAM was not filtered in any way, e.g. by removing duplicates.  
 
+[ what about putting quickstart/examples/how2run here making it easier to see (my personal preference)]
+
 ## Remarks
 
-  * Paired-end FASTQs are done with biobambam2's bamtofastq
+  * Paired-end FASTQs are generated with biobambam2's bamtofastq
   * Sorting is done with the UNIX coreutils tool "sort" in an efficient way (e.g. sorting per read-group; co-sorting of order-matched mate FASTQs).
-  
+
 ## Status
 
   * Explicit single-end BAM processing is not implemented. Possibly it works with the paired-end pipeline. The original (Roddy) Picard-based code was left in the workflow, but was never tested in this workflow.
-  * This is (yet) really more of an exercise to implement a Nextflow workflow. 
-  
+  * This is (yet) really more of an exercise to implement a Nextflow workflow.
+
 ## Parameters
 
-When doing BAM-to-FASTQ conversion with subsequent sorting, the unsorted output of the conversion step are called "intermediate" FASTQs. In this case, you can choose to leave the intermediate FASTQs unsorted and thus trade CPU time for IO-time. Note that the output FASTQs of the workflow (possibly unsorted if `sortFastqs == false`) are always compressed.
+When doing BAM-to-FASTQ conversion with subsequent sorting, the unsorted output of the conversion step are called "intermediate" FASTQs. In this case, you can choose to leave the intermediate FASTQs unsorted and thus trade CPU time for IO-time. Note that the output FASTQs of the workflow (possibly unsorted if `sortFastqs == false`) are always compressed [how?].
+
+**Required**
 
   * `bamFileList`: Comma-separated list of input BAM-file paths.
+  * `outputDir`: Output directory
+
+
+**Optional**
+
   * `sortFastqs`: Whether to produce FASTQs in a similar order as in the input BAM (`false`) or sort by name (`true`). Default: `true`.
   * `pairedEnd`: Data is paired-end (`true`) or single-end (`false`). Default: `true`
-  * `writeUnpairedFastqs`: Reads may have lost their mate, e.g. because of filtering or truncation of the BAM. By default, these reads are written to a `_U` file.
+  * `writeUnpairedFastqs`: Reads may have lost their mate, e.g. because of filtering or truncation of the BAM. By default, these reads are written to a `_U` file. [ what is a _U file ]
   * `excludedFlags`: Comma-separated list of flags. Default: "secondary,supplementary" (thus with complete BWA-aligned BAM input exactly the reads of the input are produced)
   * outputPerReadGroup: Whether reads from different read-groups should be written to different files. Default: true
   * `checkFastqMd5`: While reading in intermediate FASTQs, check that the MD5 is the same as in the accompanied '.md5' file. Only available for Picard, as Biobambam does not produce MD5 files for output files. Default: `true`.
```

Comment:
Removed. Only for Picard. See also https://github.com/DKFZ-ODCF/nf-bam2fastq/issues/10.